### ISSUE TITLE
[Request] quasar init -c (extremely low priority)

### ISSUE THREAD

**Issue by rossity** [quasarframework/quasar-cli#138]
Hello,

I would love to have the option to have the option to initialize a quasar project in the current folder. 

I'm on windows and my current workflow looks like this:
1. create `project-directory` in development directory
2. right-click `project-directory` and open in vscode
3. open vscode terminal
4. initialize whatever project I'm working on

With quasar:
1. open _windows terminal_ in development directory
2. quasar init `project-directory`
3. right-click `project-directory` and open in vscode
4. close _windows terminal_

Basically it would be cool to do this all in vscode without touching a separate terminal so I'm wondering if we can have a `quasar init -c` that initiates a project in the current directory instead of having to specify a name.

**Comment by rstoenescu** [comment]
Hi, try with "quasar init ." where the dot is any console refers to the current project.
